### ISSUE TITLE
get_defined_vars Error

### ISSUE THREAD

**Issue by PwrSrg** [bkdotcom/PHPDebugConsole#19]
When I pass in get_defined_vars() I receive the following error:
"Object of class bdk\Debug could not be converted to string"

**Comment by bkdotcom** [comment]
are you getting the error when doing something like the following?

```PHP
$debug = new \bdk\Debug();

$foo = 'foo';
$bar = 'bar';
$debug->log('defined vars', get_defined_vars());
```

**Comment by PwrSrg** [comment]
Yes, that is correct.

$debug->info('Defined Vars', get_defined_vars());

**Comment by bkdotcom** [comment]
This has been fixed in the master branch (upcoming v2.3 release)

hopefully released soon (primarily working on documentation at this point)

give it a try if your able

Thanks!

(I'll close this issue when v2.3 is released)

**Comment by bkdotcom** [comment]
sorry for the delay.  
2.3 is finally out  
I hope to be more responsive with point-releases in the future
